tests/Unit/PrivateKeyConstructTest.php

```diff
@@ -61,4 +61,12 @@ public function testConstructWithInvalidButBase64Content(): void
         $this->expectExceptionMessage('Private key is not PEM');
         new PrivateKey('INVALID+CONTENT', '');
     }
+    
```

Comment:
Para los test que fallan al insertar este cambio, por compatibilidad con la versión actual, se podría:
- cambiar los test para que ahora no retorne la excepción UnexpectedValueException "Private key is not PEM" y esperar solamente RuntimeException "Cannot open private key"
- agregar en callOnPrivateKey un segundo parámetro opcional con la excepción a devolver si no se puede abrir la llave, y llamarla desde el contructor con UnexpectedValueException "Private key is not valid"

Reply:
Creo que la primera opción es mejor. Hice una prueba de concepto de la segunda opción y el problema es que ahora la excepción RuntimeException "Cannot open private key" nunca se lanzaría.

Reply:
y como se le hace para continuar, hago un nuevo commit? y se continua con este pull request?
o se hace un nuevo pull request?
Saludos.